R/token-define.R

```diff
@@ -1,31 +1,32 @@
-token <- tibble::tribble(
-  ~text,           ~class,         ~token,
-  "&",          "logical",          "AND",
-  "&&",         "logical",         "AND2",
-  "|",          "logical",           "OR",
-  "||",         "logical",          "OR2",
-  ">",          "logical",           "GT",
-  "<",          "logical",           "LT",
-  "<=",         "logical",           "LE",
-  ">=",         "logical",           "GE",
-  "!=",         "logical",           "NE",
-  "==",         "logical",           "EQ",
-  "=",      "assign_left",       "EQ_SUB",
-  "=",      "assign_left",    "EQ_ASSIGN",
-  "<-",     "assign_left",  "LEFT_ASSIGN",
-  "->",    "assign_right", "RIGHT_ASSIGN",
-  "+",             "math",          "'+'",
-  "-",             "math",          "'-'",
-  "*",             "math",          "'*'",
-  "/",             "math",          "'/'",
-  "^",             "math",          "'^'",
-  "~",          "formula",          "'~'",
-  "if",            "cond",           "IF",
-  "else",          "cond",         "ELSE",
-  "in",       "loop_cond",           "IN",
-  "while",    "loop_cond",        "WHILE"
+token <- rbind.data.frame(
```

Comment:
Another option would be `as.data.frame(matrix(ncol = 3, byrow = TRUE, c(...)))`

Reply:
Thanks for the suggestion. But I think I will just retain it like this for now.